www/js/controllers/reality-check.controller.js

```diff
@@ -0,0 +1,132 @@
+angular
+	.module('binary')
+	.controller('RealityCheckController',
+		function($scope, $rootScope, $state, $timeout, $interval, websocketService, appStateService, accountService, $ionicPopup, alertService, localStorageService) {
+			var show,
+				myInterval,
+				i = 0,
+				_landing_company_name_s,
+				myflag = 0,
+				myRequestFlag = 0;
+
+			var auth = function auth() {
+				if (!appStateService.isLoggedin) {
+					$timeout(auth, 500);
+				} else {
+					$scope.init();
+				}
+			};
+
+			$scope.init = function() {
+				$scope.$on('authorize', function(e, authorize) {
```

Comment:
since you use `appStateService.isLoggedin` to fire the init then it probably, you will lost the first authorization event broadcasting because you invoke the init after logged in. 
IMHO,  you should change you algorithm.